### ISSUE TITLE
Newtonsoft.Json.JsonSerializationException

### ISSUE THREAD

**Issue by MahesRam** [snowflakedb/snowflake-connector-net#30]
I created the json file and copied and pasted the values and changed them. Getting error 
Failed   testArrayBind
Error Message:
 OneTimeSetUp: Newtonsoft.Json.JsonSerializationException : Error converting value "<my account name>" to type 'Snowflake.Data.Tests.TestConfig'. Path 'SNOWFLAKE_TEST_USER', line 2, position 35.
  ----> System.ArgumentException : Could not cast or convert from System.String to Snowflake.Data.Tests.TestConfig.

**Comment by howryu** [comment]
Sorry. There is a mis documentation, if you want to run tests, the paramters.json file should be like this.

```
{
  "testconnection": {
    "SNOWFLAKE_TEST_USER": "TEST_DOTNET",
    "SNOWFLAKE_TEST_PASSWORD": "XXXXXXXX",
    "SNOWFLAKE_TEST_ACCOUNT": "sfctest0",
    "SNOWFLAKE_TEST_WAREHOUSE": "TESTWH_DOTNET",
    "SNOWFLAKE_TEST_DATABASE": "TESTDB_DOTNET",
    "SNOWFLAKE_TEST_SCHEMA": "TESTSCHEMA_DOTNET",
    "SNOWFLAKE_TEST_ROLE": "TESTROLE_DOTNET"
  }
}
```

Note that, there is a `testconnection` key exists. I will update the doc as well.

**Comment by howryu** [comment]
Fixed in acc5e9340f55cce938b674ec4e0f8a0e5f3bf10b
